samples/notebook_instance/my-notebook-instance.yaml

```diff
@@ -0,0 +1,17 @@
+
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: NotebookInstance
+metadata:
+  name: my-notebook
+spec:
+  instanceType: ml.t2.medium
+  notebookInstanceName: my-notebook
+  roleARN: <YOUR SAGEMAKER ROLE ARN>
+  volumeSizeInGB: 6  
+  directInternetAccess: Enabled
+  rootAccess: Enabled
+  acceleratorTypes:
+    - ml.eia1.medium
+  lifecycleConfigName: sample-lifecycle
```

Comment:
is this something you need to create separately ? If yes, could you mention in the README.

Reply:
mentioned it in create

Reply:
Moved this here https://github.com/aws-controllers-k8s/sagemaker-controller/pull/81